components/Header/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import { useTheme } from 'next-themes';
+import Link from 'next/link';
+import Logo from './logo';
+import { Button } from '@moai/core';
+import { HiOutlineMoon, HiOutlineSun } from 'react-icons/hi';
+
+const Header = () => {
+  const [mounted, setMounted] = useState(false);
+  const { theme, setTheme } = useTheme();
+
+  useEffect(() => setMounted(true), []);
+
+  const onToggleTheme = () => {
+    setTheme(theme === 'light' ? 'dark' : 'light');
+  };
+
+  if (!mounted) return null;
+
+  return (
+    <header>
+      <div className="header-content">
+        <Link href="/" passHref>
+          <a>
+            <Logo />
+          </a>
+        </Link>
+        <Button
+          onClick={onToggleTheme}
+          icon={theme === 'light' ? HiOutlineMoon : HiOutlineSun}
+          iconLabel="Đổi chủ đề"
```

Comment:
IMO, it should be "Đổi chủ đề giao diện" for better meaning. If there is no "giao diện" in this context, user may misunderstand the meaning of button.

It maybe better if blind user cannot tab to focus on it because they do not need to change theme. But Idk if Moai support setting tabindex in button or not, please check.

@huytd can you give me your opinion in this case?

Reply:
IMHO, use explicit mode name for each case based on the `theme` variable, it would be:
- "Bật chế độ ban đêm" or "Dùng giao diện tối"
- "Bật chế độ ban ngày" or "Dùng giao diện sáng"

Reply:
Sound good. I will do it